app/static/js/initialize_map.js

```diff
@@ -0,0 +1,155 @@
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+//Initial map center coordinates
+INITIAL_CENTER_LAT = 39.952;
+INITIAL_CENTER_LONG = -75.195;
+initial_coords = new google.maps.LatLng(INITIAL_CENTER_LAT, INITIAL_CENTER_LONG);
+//Initial map zoom value
+ZOOM = 17;
+//An integer for the offset between the actual window size and the
+//JavaScript-determined Window size in order to size the map
+WINDOW_OFFSET = 70
+
+//Bounds for the Date Slider. End date is current date.
+var BOUNDS_MIN = new Date(2015, 0, 1);
```

Comment:
The min date should depend on the earliest date from the incident reports, and if null, to (2015, 1, 1)